pom.xml

```diff
@@ -345,7 +311,89 @@
             </plugin>
-
         </plugins>
     </build>
+    <profiles>
+        <!-- The Configuration of the integration-test profile -->
+        <profile>
+            <id>integrationTest</id>
+            <properties>
+                <skipUTs>true</skipUTs>
+            </properties>
+            <activation>
+                <activeByDefault>false</activeByDefault>
+            </activation>
+            <build>
+                <plugins>
+                    <plugin>
+                        <groupId>org.apache.maven.plugins</groupId>
+                        <artifactId>maven-failsafe-plugin</artifactId>
+                        <version>2.22.0</version>
+                        <configuration>
+                            <skipTests>${skipTests}</skipTests>
+                            <skipITs>${skipITs}</skipITs>
+                            <systemPropertyVariables>
+                                <rabbitmq.host>localhost</rabbitmq.host>
+                                <rabbitmq.port>5672</rabbitmq.port>
+                                <rabbitmq.username>myuser</rabbitmq.username>
+                                <rabbitmq.password>myuser</rabbitmq.password>
+                                <rabbitmq.exchange>ei-exchange</rabbitmq.exchange>
+                                <rabbitmq.key>#</rabbitmq.key>
+                            </systemPropertyVariables>
+                            <excludes>
+                                <exclude>**/systemtest/**</exclude>
+                            </excludes>
+                        </configuration>
+                        <executions>
+                            <execution>
+                                <id>integration-test</id>
+                                <goals>
+                                    <goal>integration-test</goal>
+                                    <goal>verify</goal>
+                                </goals>
+                                <phase>integration-test</phase>
+                            </execution>
+                        </executions>
+                    </plugin>
+                </plugins>
+            </build>
+        </profile>
+        <!-- The Configuration of the system-test profile -->
+        <profile>
+            <id>systemTest</id>
+            <properties>
+                <skipUTs>true</skipUTs>
+            </properties>
+            <activation>
+                <activeByDefault>false</activeByDefault>
+            </activation>
+            <build>
+                <plugins>
+                    <plugin>
+                        <groupId>org.apache.maven.plugins</groupId>
+                        <artifactId>maven-failsafe-plugin</artifactId>
+                        <version>2.22.0</version>
+                        <configuration>
+                            <systemPropertyVariables>
+                                <ei.config.properties.file.path>${systemtestsProperties}</ei.config.properties.file.path>
```

Comment:
remove the extra s from systemtestsProperties. Better to have systemtest in every place we have configuration. Less confusing :)

Reply:
ok